recipes/trianglesolver/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "trianglesolver" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/trianglesolver-{{ version }}.tar.gz
+  sha256: 4af18aade579d5c0d64389b3e65aeaf06cff26319762ccd859e3268559a76aea
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python
```

Comment:
Thanks, added - although that conversation doesn't read to have concluded with this, it reads that it would be preferred to handle in the pinning metadata (also unclear whether it is suggesting pinning host to exact or minimum),

Reply:
The linter finally woke up, though.